src/Security/MemberAuthenticator/CookieAuthenticationHandler.php

```diff
@@ -27,10 +27,13 @@ class CookieAuthenticationHandler implements AuthenticationHandler
      */
     private $tokenCookieName;
 
+    private bool $tokenCookieSecure = true;
+
     /**
-     * @var boolean
+     * The SameSite value to use for authentication cookies.
+     * If set to an empty string, the default value from Cookie.default_samesite will be used.
      */
-    private $tokenCookieSecure = false;
+    private string $tokenCookieSameSite = Cookie::SAMESITE_STRICT;
```

Comment:
In https://docs.silverstripe.org/en/6/developer_guides/security/secure_coding/#secure-sessions-and-cookies when talking about updating the session cookie security it says

> The same treatment should be applied to the cookie responsible for remembering logins across sessions

For this reason I've chosen to do exactly that - I've updated its default secure and samesite values to match the session cookie values.